secedgar/cli.py

```diff
@@ -8,8 +8,14 @@
 
 
 @click.group()
-def cli():
+@click.option('--user-agent',
```

Comment:
The `--user-agent` options do not currently have any shorthand. I am thinking of going with `-u`?